Portfolio2.css

```diff
@@ -0,0 +1,210 @@
+* {
+	margin:0;
+	padding: 0;
+	
+}
+
+body {
+	background-image: url("https://wallpaperaccess.com/full/1128684.jpg");
+	background-size: cover;
+	
+	
+ }
+
+header {
+	position: fixed; 
+	top:0;
+	right:0;
+}
+
+header ul li {
+	display: inline-block;
+	list-style: none;
+	cursor:pointer;
+	padding:15px;
+	
+}
+
+.logo img {
+	width: 90px;
+	
+}
+
+.active {
+	color: grey
+	
+}
+
+#headertwo {
+    	text-align: right;
+	font-size: 50px;
+	margin-top: 200px;
+	font-family: garamond;
+  	color: #7c7b85;
+	padding: 14px 16px;
+  	text-decoration: none;
+	margin: 0 auto;
+	width:90%;
+	
+  	text-align: left;
+  	font-weight: bold;
+	  
+}
+
+#header h1 {
+	margin-left:50px;
+	
+}
+
+
+
+.sub-title {
+	width: 100%;
+	float: left;
+	}
+
+p {
+	width: 900px;
+	font-family: "Times New Roman";
+	font-size:20px;
+	line-height:25px;
+	color: white;
+	margin-right: 70px;
+}
+
+
+	
+
+.column {
+	overflow: auto;
+}
+
+.threecolumn {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	margin-bottom: 100px;
+	margin: 0 auto;
+	width:90%;
+	
+}
+
+h2 {
+	text-align: center;
+	width: 90%;
+	font-size: 40px;
+	color: white;
+	margin-top: 100px;
+}
+
+.fourcolumns {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-rows: 1fr 1fr;
+	margin-bottom: 100px;
+	margin-left: 100px;
+	
+}
+
+.twocolumns{
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	margin: 0 auto;
+	width:90%;	
+}
+.selfie  {
+	background-image:url("selfie2.jpeg");
+	background-size: cover;
+	width:300px;
+	height:330px;
+	max-width: 400px;
+}
+
+.mayden {
+	background-image:url("mayden.jpg");
+	
+}
+
+.pilotscreenshot {
+	background-image:url("pilotscreenshot.png");
+	background-size: cover;
+	width:760px;
+	height:400px;
+}
+.trillville {
+	background-image:url("trillville.jpeg");
+	background-size: cover;
+	width:760px;
+	height:400px;
+}
+
+.manu {
+	background-image:url("manu.jpg");
+	background-size: cover;
+	width:760px;
+	height:460px;
+	justify-content: center;
+}
+.rolls {
+	background-image:url("rolls.jpg");
+	background-size: cover;
+	width:760px;
+	height:460px;
+	justify-content: center;
+}
+.banner {
+	padding: 5px;
+	width: 90%;
+	margin: 0 auto;
+	background-color: #92a6a6;
+	color: #2a324b;
+	border-style: solid;
+	border-width: 1px;
+	border-color: #2a324b;
+	text-align: center;
+	margin-bottom: 30px;
+}
+
+.aboutme {
+	margin-top: 50px;
+}
+
+
+.contact {
+	margin-bottom: 10px;
+	margin-top: 50px;
+}
+
+
+
+h3 {
+	text-align: center;
+	color: grey;
+	font-size:30px;
+	margin:0 auto;
+	margin-bottom:100px;
+}
+
+a {
+	display: grid;
+	margin-top: 100px;
+	margin: 0 auto;
+	
+}
+
+@media screen and (max-width: 900px) {
+	.threecolumn {
+		display: grid;
+		grid-template-columns: 1fr ;
+	}
+}
+
+@media screen and (max-width: 600px) {
+	.fourcolumns {
+		display: grid;
+		grid-template-columns: 1fr;
+	}
+	@media screen and (max-width: 600px) {
+		.twocolumns {
+			display: grid;
+			grid-template-columns: 1fr;
+		}
```

Comment:
These media queries work well, but I think you also need to adjust things like your image sizes on mobile, as for me on an iPhone x they are wider than the page. Try using the device mode in the chrome dev tools (the little phone and tablet icon on the top left) so you can see what I mean